components/input/Input.tsx

```diff
@@ -293,6 +295,7 @@ class Input extends React.Component<InputProps, InputState> {
           },
         )}
         ref={this.saveInput}
+        {...dataOrAriaProps}
```

Comment:
Same as Button. Input already pass the `otherProps` in. No need to get additional logic of this.